### PR TITLE
fix: e2e test with sql runner and csv download

### PR DESCRIPTION
**Pull request by IrakliJani**
this should fix the e2e test we are having with csv downloads 🍀

### CODE REVIEW

**Comment by rephus** [comment]
AHh, thanks, this was failing on my PR

**Review by rephus — APPROVED** [review]
if e2e work, then we can merge

**Comment by owlas** [comment]
Looks like the error is still there

File: packages/e2e/cypress/e2e/downloadCsv.cy.ts [hunk 1]
```diff
@@ -130,7 +130,6 @@ describe('Download CSV on Explore', () => {
         cy.findByText('Loading chart').should('not.exist');
 
         cy.findByText('Export CSV').click();
-        cy.get('.bp4-popover2').contains('button', 'Export CSV').click();
```
Comment (ZeRego) [inline packages/e2e/cypress/e2e/downloadCsv.cy.ts:133]: The e2e test is still failing on the next line. I suggest we increase the timeout to 10 seconds. 🤔

**Pull request merged**